### PR TITLE
mgmt, cognitiveservices, add live tests

### PR DESCRIPTION
**Pull request by v-hongli1**
relate to https://github.com/Azure/azure-sdk-for-java/issues/35316

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/cognitiveservices/azure-resourcemanager-cognitiveservices/src/test/java/com/azure/resourcemanager/cognitiveservices/CognitiveServicesManagerTests.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    public void testCreateAccount() {
+        Account account = null;
+        try {
+            String accountName = "account" + randomPadding();
+            // @embedmeStart
+            account = cognitiveServicesManager.accounts()
+                .define(accountName)
+                .withExistingResourceGroup(resourceGroupName)
+                .withRegion(REGION)
+                .withKind("CognitiveServices")
```
Comment (weidongxu-microsoft) [inline sdk/cognitiveservices/azure-resourcemanager-cognitiveservices/src/test/java/com/azure/resourcemanager/cognitiveservices/CognitiveServicesManagerTests.java:84]: Does backend really need this?

Comment (v-hongli1) [reply sdk/cognitiveservices/azure-resourcemanager-cognitiveservices/src/test/java/com/azure/resourcemanager/cognitiveservices/CognitiveServicesManagerTests.java:80]: Redundant parameter settings have been removed in the new version.

Comment (XiaofeiCao) [reply sdk/cognitiveservices/azure-resourcemanager-cognitiveservices/src/test/java/com/azure/resourcemanager/cognitiveservices/CognitiveServicesManagerTests.java:80]: Is `withKind("CognitiveServices")` required?

Comment (v-hongli1) [reply sdk/cognitiveservices/azure-resourcemanager-cognitiveservices/src/test/java/com/azure/resourcemanager/cognitiveservices/CognitiveServicesManagerTests.java:80]: I tried to remove `withKind("CognitiveServices")`, but an exception occurs.
image
So it is required.

**Review by weidongxu-microsoft — APPROVED** [review]

File: sdk/cognitiveservices/azure-resourcemanager-cognitiveservices/src/test/java/com/azure/resourcemanager/cognitiveservices/CognitiveServicesManagerTests.java (hunk 1 above)
Comment (weidongxu-microsoft) [reply sdk/cognitiveservices/azure-resourcemanager-cognitiveservices/src/test/java/com/azure/resourcemanager/cognitiveservices/CognitiveServicesManagerTests.java:80]: Thanks. Crazy backend, as if it does not know what service it is...

**Review by XiaofeiCao — APPROVED** [review]

**Pull request merged**